### PR TITLE
Themed components

### PR DESCRIPTION
**Pull request by Jashpatel1**

### CODE REVIEW

File: frontend/src/components/Footer.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template lang="pug">
-    v-footer(color="#272727" padless)
+    v-footer(color="#1b65c4" padless)
```
Comment (Tanuj22) [inline frontend/src/components/Footer.vue]: Hardcoded Color

File: frontend/src/components/ProfileCard.vue [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  v-card(v-for="Profile in Profiles" :key="Profile.name" :width="1.05*getProfileWidth")
     v-stepper(v-model="e1")
       v-stepper-items
-        v-stepper-content.pa-1(step="1")
-          v-card.pa-1(:width="getProfileWidth")
+        v-stepper-content.pa-0(step="1")
+          v-card.pl-3(:width="getProfileWidth" flat)
             v-row 
-              v-col(cols="4") 
+              v-col(cols="4" style="background-color:#1b65c4")
```
Comment (Tanuj22) [inline frontend/src/components/ProfileCard.vue:9]: Hardcoded Color

File: frontend/src/components/ProfileCard.vue [hunk 3]
```diff
@@ -24,12 +24,12 @@ v-container
                     v-icon(color="orange") mdi-gitlab
                   v-btn(:href="Profile.LinkedinLink" icon small v-if="Profile.LinkedinLink")
                     v-icon(color="blue") mdi-linkedin     
-              v-col.pa-1(cols="2")    
-                v-icon(@click="e1 = 2") mdi-arrow-right-box
+              v-col.pt-1.pa-0(cols="1")    
+                v-icon(@click="e1 = 2" color="#1b65c4") mdi-arrow-right-box
```
Comment (Tanuj22) [inline frontend/src/components/ProfileCard.vue]: Hardcoded Color

File: frontend/src/components/ProfileCard.vue [hunk 4]
```diff
@@ -24,12 +24,12 @@ v-container
...
                     v-icon(color="blue") mdi-linkedin     
-              v-col.pa-1(cols="2")    
-                v-icon(@click="e1 = 2") mdi-arrow-right-box
+              v-col.pt-1.pa-0(cols="1")    
+                v-icon(@click="e1 = 2" color="#1b65c4") mdi-arrow-right-box
         v-stepper-content.pa-1(step="2")
           v-row
             v-col(cols="1")
-              v-icon(@click="e1 = 1") mdi-arrow-left-box
+              v-icon(@click="e1 = 1" color="#1b65c4") mdi-arrow-left-box
```
Comment (Tanuj22) [inline frontend/src/components/ProfileCard.vue]: Hardcoded Color

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Footer.vue (hunk 1 above)
Comment (Jashpatel1) [reply frontend/src/components/Footer.vue]: done

File: frontend/src/components/ProfileCard.vue (hunk 2 above)
Comment (Jashpatel1) [reply frontend/src/components/ProfileCard.vue:9]: done

File: frontend/src/components/ProfileCard.vue (hunk 4 above)
Comment (Jashpatel1) [reply frontend/src/components/ProfileCard.vue]: done

File: frontend/src/components/ProfileCard.vue (hunk 3 above)
Comment (Jashpatel1) [reply frontend/src/components/ProfileCard.vue]: done

**Pull request merged**